### DISCUSSION TITLE
Maven artifacts for 1.1.0

### DISCUSSION THREAD

**Discussion by ladinu** [apache/pekko-grpc#384] [Q&A] [answered]
Hi, I noticed 1.1.0 was released recently, but not maven artifacts. Is there a process to request publishing of maven artifacts?

**Comment by pjfanning** [comment, marked as answer]
No release has been announced. The release is still in progress.

All release related items are discussed on the mailing list and that can be viewed at https://lists.apache.org/list.html?dev@pekko.apache.org

You will find a mail thread there about releasing the jars.

The announcement mail will also be sent to that list when the release is complete.

**Reply by ladinu** [reply]
Thanks! Just subscribed to the list
